opds_import.py

```diff
@@ -727,7 +750,7 @@ def _parse_identifier(self, identifier):
         """
         return parse_identifier(self._db, identifier)
 
-    def import_from_feed(self, feed, feed_url=None):
+    def import_from_feed(self, feed, feed_url=None, custom_identifier=None):
```

Comment:
Other pieces of configuration like `data_source_name` and `mirrors` are configured in the `OPDSImporter` constructor; this should be moved to the constructor as well. (And it should be called something like `primary_identifier_source` -- same ideas as my earlier comment.)

Reply:
I was trying to do this, but I just use the same logic of the custom accept header. I'm changing to use the same logic of `data_source_name` and `mirrors` now.

Reply:
I see. Since the `OPDSImporter` constructor takes a `collection` as an argument, you can pull the information from the collection configuration, as you do in `OPDSImportMonitor` with the custom accept header, and avoid adding any new arguments to any method signatures.